src/app/legacy/containers/CpsRecommendations/RecommendationsPromo/index.jsx

```diff
@@ -138,7 +143,12 @@ const RecommendationsPromo = ({ promo, eventTrackingData }) => {
 };
 
 RecommendationsPromo.propTypes = {
-  promo: oneOfType([shape(storyItem)]).isRequired,
+  promo: oneOfType([
+    shape(storyItem),
+    shape(cpsAssetPagePromoPropTypes),
+    shape(optimoPromoPropTypes),
+    shape(mediaPromoPropTypes),
```

Comment:
Not 100% sure if this is right... but I don't believe there's another way of suppressing the error...